src/sidebars/favourites/favourite-controller.js

```diff
@@ -66,6 +67,12 @@
 
                 this.favourites = this.store.get();
 
+                if (this.favourites.length === 0) {
+                    this.$timeout(() => {
+                        this.receivedFirstQuandlResponse = true;
+                    });
+                }
+
```

Comment:
In case there's a window with no favourites in storage. This is possible, but probably another issue's scope.